scripts/HPC_chatbot_preprocessor/parsed_mds/generic/account/account_paragraph_2.txt

```diff
@@ -0,0 +1,9 @@
+The web portal offers a convenient way to upload files and gain shell access to the HPC-UGent infrastructure from a standard web browser (no software installation or configuration required).
+If you would like use a terminal with SSH as this gives you more flexibility continue reading.
+However if you prefer to use the web portal, you can skip ahead to the following section: Applying for the account.
+Once you have successfully obtained an account, you can then delve into the details of utilizing the HPC-UGent web portal by reading Using the HPC-UGent web portal.
```

Comment:
is it possible to provide some auto mated way to convert the reference to the section with something more clear:

eg `... by reading Using ...` -> `... by reading section "Using ..."` (not sure what the impact of the quotes is; possibly nothing, but the section might help).